### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by rahulcs754**
Added avatar css

https://user-images.githubusercontent.com/6254980/151846920-b311ad3e-fb67-4024-b827-c73fe364aad1.mp4

Please check and preview

### CODE REVIEW

File: css/components/avatar.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
+/*
+|| avatar css
+*/
+
+/* Main Avatar */
+.avatar{
+    display: flex;
+	align-items: center;
```
Comment (coldpigli) [inline css/components/avatar.css]: any specific reason for indenting align-items and justify-content?
They can be in the same indentation level as the rest.

Comment (rahulcs754) [reply css/components/avatar.css]: you are right there is no need to align-items and justify-content 
i removed both of line with display flex

Comment (rahulcs754) [reply css/components/avatar.css]: i removed it

File: css/components/avatar.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+.avatar-square,
+.avatar-img-square{
+    border-radius: 10%;
+}
+
+.avatar-circle,
+.avatar-img-circle{
+    border-radius: 50%;
+}
+
```
Comment (tanishq20) [inline css/components/avatar.css]: Avoid giving extra spaces.

**Review by tanishq20 — COMMENTED** [review]
Code looks fine.

-  suggested a small change
-  solve the branch conflict arised

File: css/components/avatar.css (hunk 2 above)
Comment (rahulcs754) [reply css/components/avatar.css]: removed

Comment (tanishq20) [reply css/components/avatar.css]: Okay👍

**Review by tanishq20 — APPROVED** [review]

**Review by coldpigli — APPROVED** [review]

**Comment by rahulcs754** [comment]
finally merge

**Pull request merged**